src/fparser/two/parser.py

```diff
@@ -223,7 +223,7 @@ def _rpl_list(clsname):
                     return []
                 # remove this code when all classes are implemented.
                 cls = base_classes[clsname]
-                if 'match' in cls.__dict__:
+                if hasattr(cls, 'match'):# in cls.__dict__:
```

Comment:
I'm happy with this change so if you could just remove the commented-out old form that will be great.

Reply:
Sorry, that was an oversight. Thanks for spotting!